packages/mantine/src/widgets/PasswordWidget.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback } from 'react';
+import {
+  ariaDescribedByIds,
+  FormContextType,
+  labelValue,
+  RJSFSchema,
+  StrictRJSFSchema,
+  WidgetProps,
+} from '@rjsf/utils';
+import { PasswordInput } from '@mantine/core';
+import { cleanupOptions } from '../utils';
```

Comment:
Please add blank line between system and local imports
```suggestion
import { PasswordInput } from '@mantine/core';

import { cleanupOptions } from '../utils';
```